include/utils.h

```diff
@@ -30,6 +30,7 @@
 #include <inttypes.h>
 #include <stdio.h>
 #include <stdint.h>
+#include <netinet/in.h>
```

Comment:
This requires ifdef, for Win32, at least. (So, probably `if defined (__unix__) || defined (__APPLE__)` or something)